setup.py

```diff
@@ -9,6 +9,7 @@
 
 from setuptools import setup, find_packages
 
+
 CLASSIFIERS = [
     'Development Status :: 5 - Production/Stable',
     'Intended Audience :: Developers',
```

Comment:
Could you please add PyPI classifiers for supported Python versions after the `Programming Language :: Python` one?

```
'Programming Language :: Python :: 2.7',
'Programming Language :: Python :: 3.6',
'Programming Language :: Python :: 3.7',
'Programming Language :: Python :: 3.8',
'Programming Language :: Python :: 3.9',
```